src/Parsers/IParser.h

```diff
@@ -49,6 +49,13 @@ struct Expected
 };
 
 
+/** Parser settings (for compatibility mostly)
+  */
+struct ParserSettings
+{
+    bool parser_settings_after_format_compact = false;
```

Comment:
It seems the name `allow_settings_after_format_in_insert` is shorter and clearer.

Reply:
Yep, thanks, will rename.